packages/neos-ui/src/manifest.dataloaders.js

```diff
@@ -221,7 +221,7 @@ manifest('main.dataloaders', {}, globalRegistry => {
             }).then(() => {
                 // Trigger query
                 const assetProxySearchApi = backend.get().endpoints.assetProxySearch;
-                const resultPromise = assetProxySearchApi(searchTerm, '', options);
```

Comment:
Why can this be removed?

Reply:
I moved the "assetSourceIdentifier" to the options (and it supports multiple asset source constraints now). See https://github.com/neos/neos-ui/pull/2759/files#diff-d34d33e9dc47b1d77ad5f29e5a091011R260